docs/_layouts/v2.html

```diff
@@ -120,12 +122,14 @@ <h1><a href="https://leafletjs.com/"><img src="{{ root }}docs/images/logo.png" a
 	<div class="footer">
 		<p>&copy; 2010–{{ 'now' | date: "%Y" }} <a href="https://agafonkin.com">Volodymyr Agafonkin</a>. Maps &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors.</p>
 	</div>
-
 	<nav class="ext-links">
-	  <a class="ext-link twitter" href="https://twitter.com/LeafletJS" title="Follow LeafletJS on Twitter"><img alt="Follow LeafletJS on Twitter" src="{{root}}docs/images/twitter-round.svg" width="46" /></a>
+		<a class="ext-link twitter" href="https://twitter.com/LeafletJS" title="Follow LeafletJS on Twitter"><img alt="Follow LeafletJS on Twitter" src="{{root}}docs/images/twitter-round.svg" width="46" /></a>
 	  <a class="ext-link github" href="http://github.com/Leaflet/Leaflet" title="View Source on GitHub"><img alt="View Source on GitHub" src="{{root}}docs/images/github-round.svg" width="46" /></a>
 	  <a class="ext-link forum" href="https://stackoverflow.com/questions/tagged/leaflet" title="Ask for help on Stack Overflow"><img alt="Leaflet questions on Stack Overflow" src="{{root}}docs/images/forum-round.svg" width="46" /></a>
+		<div id="backtotop" class="ext-link"><div class="backtotop"></div></div>
```

Comment:
I think it makes sense, to have this not in this `<nav>` element, because it is displayed at the bottom

Reply:
Yeah! Before it was just next to the nav elements. And then I re positioned it to the right bottom. I will re-position it to somewhere else.